src/lib/rest/rest.cpp

```diff
@@ -193,6 +193,21 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 
   LM_T(LmtUriParams, ("URI parameter:   %s: %s", key.c_str(), ciP->uriParam[key].c_str()));
 
+  //
+  // Now check the URI param has no invalid characters
+  // Except for the URI param 'q' that is not to be checked for invalid characters
+  //
+  if (key != "q")
+  {
+    if (forbiddenChars(ckey) || forbiddenChars(val))
```

Comment:
Yes, I was surprised to find out we didn't have any check of illegal chars in URI params ...